src/store/flow/resource.ts

```diff
@@ -222,6 +222,5 @@ export function discoverContentTypesFor(mode: SupportedMode, resource?: IResourc
     }, contentTypeOverrides)
     return contentTypeOverrides
   }, contentTypeOverrides)
-
-  return Object.assign(defaultModeMappings, contentTypeOverrides)[mode]
+  return Object.assign(defaultModeMappings, contentTypeOverrides)[mode][0]
```

Comment:
this avoids us to get array inside array values like `"sms":[["text"]]`, and give `"sms":["text"]`

Reply:
@bzabos  , for info: this seems fixing the resources issue for **new added blocks** (except for LogBlock). However, existing blocks (created from `2019-10-10-shortcut-flow.json`) have resources broken. Maybe a clue for https://viamoinc.atlassian.net/browse/VMO-2671

Reply:
I removed this change in this current PR